charts/consul/templates/client-daemonset.yaml

```diff
@@ -231,13 +231,16 @@ spec:
             {{- if .Values.global.tls.enabled }}
             - name: CONSUL_HTTP_ADDR
               value: https://localhost:8501
-            {{- if .Values.global.tls.enableAutoEncrypt }}
+              {{- if .Values.global.tls.enableAutoEncrypt }}
```

Comment:
This `if` and `end` need to get unintended. I think they were accidentally tabbed.